ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -268,24 +270,19 @@ Set Expected Items Based On RHODS Type    # robocop: disable
     ...    Jupyter
     ${EXPECTED_ITEM_PROVIDERS}=    Create List       by Anaconda Professional
     ${EXPECTED_ITEM_RESOURCE_TYPE}=    Create List     Tutorial
-    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}=    Create List
-    ...    Creating a Jupyter notebook
-    ...    How to install Python packages on your notebook server
-    ...    How to update notebook server settings
-    ...    How to use data from Amazon S3 buckets
-    ...    How to view installed packages on your notebook server
-    ...    Deploying a sample Python application using Flask and OpenShift.
-    ...    Jupyter
```

Comment:
Are these changes expected? Some tracking issue or something that it has been decided intentionally?

Reply:
It's same list as before (just sorted as seen on UI)